frontend/css/style.css

```
@@ -4555,939 +4561,20 @@ input:checked+.slider {
-    background: var(--wood-light);
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    color: rgba(255, 255, 255, 0.6);
-    font-size: 1.5rem;
-    box-shadow: 3px 3px 8px rgba(0, 0, 0, 0.15);
+html[data-theme="night"] .glass-overlay,
+body.night-mode .glass-overlay,
+[data-theme="night"] .glass-overlay {
+    background-color: #1e140c !important;
+    background: #1e140c !important;
```

Stylelint (17.11.0)

[error] Overridden property "background-color" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)